ui/components/multichain/pages/page/components/header/header.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { HeaderBase, Text } from '../../../../../component-library';
+import {
+  BlockSize,
+  TextAlign,
+  TextVariant,
+} from '../../../../../../helpers/constants/design-system';
+
+export const Header = ({
+  children,
+  endAccessory = null,
+  startAccessory = null,
+  className = '',
+  ...props
+}) => {
+  return (
+    <HeaderBase
+      padding={4}
+      width={BlockSize.Full}
+      className={classnames('multichain-page-header', className)}
+      startAccessory={startAccessory}
+      endAccessory={endAccessory}
+      {...props}
```

Comment:
I think we'll need the wrapperProps from the HeaderBase, unless there's another way to enforce styling. While testing this with connections, I ran into the overflow issue again with long text in the children prop. It was fixed in #21217, if that helps at all.

Reply:
Overflow fixed here:  https://github.com/MetaMask/metamask-extension/pull/21245/commits/2611d13ec7db255b1b3cf6568281203c052ee7c5